LSP.sublime-settings

```diff
@@ -120,6 +120,10 @@
   // sorting algorithm and instead uses the sorting defined by the relevant language server.
   "inhibit_word_completions": true,
 
+  // Controls whether words are overwritten when accepting completions.
+  // Note that a language server needs to support this feature to work.
```

Comment:
>  // Note that the **"replace"** mode only works if the server supports it. Otherwise the mode fall backs to "insert".

The sentence at the end is not entirely correct. I would just replace `"replace"` with  "completion_insert_mode". and delete the second sentence.

>  // Note that the **"completion_insert_mode"** will only works if the server supports it.

Reply:
But maybe we should think how can we simplify this option for the average joe.

You seem to be saying that there are basically three ways to insert completions - using the old TextEdit and then the "insert" and "replace" from the new `InsertReplaceEdit`. Isn't that a bit much to understand for the average user? Can we make the old TextEdit completion fall into the "insert" case? Otherwise I feel it would be very hard for the user to make a decision here without having deep understanding of the spec and implications of each behavior.